playbooks/install_stack.yaml

```diff
@@ -475,6 +467,13 @@
         when:
         - (tripleo_repos_branch is defined and tripleo_repos_branch not in ['train', 'ussuri', 'victoria']) or (tripleo_repos_branch is not defined)
 
+  - name: Deploy Ceph
+    include_role:
+      name: ceph
+    when:
+      - ceph_enabled
+      - ceph_adm_enabled
```

Comment:
@mdbooth in theory if rhos_release <= 17 (where there's no cephadm) should allow to skip this task, and `tripleo_operator.tripleo_deploy` should be run w/ the ceph-ansible related templates (in case ceph is enabled)